lnrpc/walletrpc/walletkit_server.go

```diff
@@ -958,6 +958,15 @@ func UnmarshallOutPoint(op *lnrpc.OutPoint) (*wire.OutPoint, error) {
 func validateBumpFeeRequest(in *BumpFeeRequest) (
 	fn.Option[chainfee.SatPerKWeight], bool, error) {
 
+	// We make sure either the conf target or the exact fee rate is
+	// specified for the starting fee of the fee function.
+	if in.TargetConf != 0 && in.SatPerVbyte != 0 {
+		return fn.None[chainfee.SatPerKWeight](), false,
```

Comment:
nit: could move this to after the definition of `satPerKwOpt` so you could use that in the return value list.